test/gpu_test.py

```diff
@@ -129,7 +140,7 @@ def test_a100_20gb_gpu_unsupported():
 
     with pytest.raises(ValueError) as err:
         stub.function(gpu=modal.gpu.A100(memory=20))(dummy)
-    assert err.value.args == ("A100 20GB is unsupported, consider A10 or A100 40GB instead",)
+    assert err.value.args[0].startswith("A100 20GB is unsupported, consider")
```

Comment:
FYI in case you didn't know, can also pass an expected error message regex to the `match` parameter of `pytest.raises`.